packages/block-library/src/buttons/editor.scss

```diff
@@ -1,18 +1,17 @@
 // This variable is repeated across Button, Buttons, and Buttons editor styles.
-$blocks-block__margin: 0.5em;
+$blocks-block__margin: 1em;
 
 .wp-block > .wp-block-buttons {
 	display: flex;
 	flex-wrap: wrap;
 }
 
 .wp-block-buttons {
+	// Specificity needed to override editor auto block margins.
 	> .wp-block {
-		// Override editor auto block margins.
-		margin-left: 0;
```

Comment:
It looks like we might still need this rule for classic themes, which sets `.editor-styles-wrapper .wp-block` to `0` for margin left and right [here](https://github.com/wordpress/gutenberg/blob/7f97d5c3da65c73ea026171e3d9335c087ab2dde/packages/edit-post/src/classic.scss#L5-L5)

Left justified Buttons block in TwentyTwentyOne:

| Removing this rule | With this rule restored / set to 0 |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/14988353/135969973-5cf5d9c3-8a80-44b5-9d5a-6a623f3fb625.png) | ![image](https://user-images.githubusercontent.com/14988353/135970003-17f0188c-b95f-4905-9aa8-d93c79289c7d.png) |

Reply:
Thanks, this should be fixed as we go back to using `gap` and setting all the margins to 0

Reply:
Unfortunately, I think we might still need this pesky left/right margin rule for classic themes. With TwentyNineteen applied, and a Buttons block set to "Justify items right", while the block is selected, the appender is rendered next to the buttons and receives the `wp-block` class. The margin-left/right from `classic.css` then creates too much room surrounding the appender, which visually makes the buttons appear as though they're left aligned:

![image](https://user-images.githubusercontent.com/14988353/136150752-bcdd293a-23a2-48c9-9626-844fa1a70752.png)

When de-selecting the Buttons block, it then appears to be aligned correctly:

![Kapture 2021-10-06 at 17 22 20](https://user-images.githubusercontent.com/14988353/136151019-b873c047-59fa-46ca-9da9-57bb6124a5b4.gif)